python-sdk/tests/sql/operators/test_snowflake_merge_func.py

```diff
@@ -44,12 +44,12 @@ def test_merge_func(self):
         assert (
             sql == "merge into IDENTIFIER(:target_table) using IDENTIFIER(:source_table) "
             "on Identifier(:merge_clause_target_0)=Identifier(:merge_clause_source_0) "
-            f"when matched then UPDATE SET {self.target_table_name}.sell={self.source_table_name}.sell "
-            f"when not matched then insert({self.target_table_name}.sell) values ({self.source_table_name}.sell)"
+            f'when matched then UPDATE SET {self.target_table_name}."sell"={self.source_table_name}."sell" '
+            f'when not matched then insert({self.target_table_name}."sell") values ({self.source_table_name}."sell")'
```

Comment:
Weren't this lower case already, do we need quotes for it?

Reply:
Removed it.